README.md

```diff
@@ -98,6 +100,34 @@ var schema = new parquet.ParquetSchema({
 });
 ```
 
+### JSON Schema
```

Comment:
Does it makes sense to provide also an example with the `create<Type>Field` functions?

With this PR we have - please correct me if I'm wrong - three options to create a schema:

* native w/o helper functions ( plain? )
* native w/ helper functions
* JSON Schema

```
var schema = new parquet.ParquetSchema({
  name: parquet.ParquetSchema.createStringField(),
  quantity: parquet.ParquetSchema.createIntField(64),
  price: parquet.ParquetSchema.createDoubleField(),
  date: parquet.ParquetSchema.createTimestampField(),
  in_stock: parquet.ParquetSchema.createBooleanField()
});
```

Reply:
Good idea. Added!